libraries/stencil/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "main": "dist/index.js",
+  "module": "dist/index.mjs",
+  "es2015": "dist/esm/index.mjs",
+  "es2017": "dist/esm/index.mjs",
+  "types": "dist/types/components.d.ts",
+  "collection": "dist/collection/collection-manifest.json",
+  "collection:main": "dist/collection/index.js",
+  "unpkg": "dist/stencil-elements-everywhere/stencil-elements-everywhere.js",
+  "scripts": {
+    "test": "cross-env LIBRARY_NAME=@stencil/core karma start",
+    "build": "stencil build --docs && npm run test"
+  },
+  "devDependencies": {
+    "@babel/core": "7.6.2",
+    "babel-loader": "8.0.6",
+    "@babel/preset-env": "7.6.2",
```

Comment:
I think the other libraries are on:

```
"babel-core": "6.26.3",
"babel-loader": "7.1.5",
"babel-preset-es2015": "6.24.1",
```

If possible, I'd suggest using those dependencies.